packages/react-reconciler/src/ReactFiberErrorLogger.js

```diff
@@ -94,13 +96,33 @@ export function defaultOnCaughtError(
       }.`;
 
     if (enableOwnerStacks) {
```

Comment:
I only added this feature to the `enableOwnerStacks` flag because we the other branch uses the uninstrumented `console['error'](...)` and we don't have a version of `printToConsole` that excludes adding a component stack. So to avoid forking all that behavior for the old branch I just made this a "new world" feature.